MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+</td>
+<td>
+Generated for TimeSpan or TimeOnly property type and properties with [DataType(DataType.Time)] attribute. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormPickerItem
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for Enum type property. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormCheckBoxItem
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the Bool type property.
+</td>
+</tr>
+<tr>
+<td>
+DataFormSwitchItem
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the Bool type property.
+</td>
+</tr>
+<tr>
+<td>
+DataFormComboBoxItem
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+<td>
+Generated for Enum type property. 
+</td>
+</tr><tr>
+<td>
+`DataFormAutoCompleteItem`
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+<td>
+Generated for Enum type property. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormRadioGroupItem
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the Enum type property.
+</td>
+</tr>
+</table>
+
+You can customize the property settings or cancel the generation of `DataFormItem` by handling the `GenerateDataFormItem` event.
+
+## Customize auto generated fields
+
+You can customize or cancel the generated `DataFormItem` by handling the `GenerateDataFormItem` event. This event occurs when the field is auto-generated for public and non-static property of the data object.
+
+{% tabs %}
+{% highlight XAML %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="GettingStarted.MainPage">
+    <ContentPage.Content>
+        <dataForm:SfDataForm x:Name="dataForm" GenerateDataFormItem="OnGenerateDataFormItem"/>
+    </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`GenerateDataFormItemEventArgs.DataFormItem`  property returns the newly created `DataFormItem`.
+`GenerateDataFormItemEventArgs.DataFormGroupItem`  property returns the newly created `DataFormGroupItem`.
+
+## Cancel DataFormItem generation of the data field
```

Comment:
## Cancel to generate data editor

You can cancel the specific data editor by handling the `GenerateDataFormItem` event or by defining display attribute to avoid the particular data field being displayed.

Reply:
title and content changed